frontend/appflowy_flutter/lib/plugins/database_view/widgets/row/row_detail.dart

```diff
@@ -43,147 +45,100 @@ class RowDetailPage extends StatefulWidget with FlowyOverlayDelegate {
 }
 
 class _RowDetailPageState extends State<RowDetailPage> {
-  final padding = const EdgeInsets.symmetric(
-    horizontal: 40,
-    vertical: 20,
-  );
-
   @override
   Widget build(BuildContext context) {
     return FlowyDialog(
       child: BlocProvider(
         create: (context) {
-          final bloc = RowDetailBloc(
-            dataController: widget.dataController,
-          );
-          bloc.add(const RowDetailEvent.initial());
-          return bloc;
+          return RowDetailBloc(dataController: widget.dataController)
+            ..add(const RowDetailEvent.initial());
         },
-        child: Padding(
-          padding: padding,
-          child: Column(
-            children: [
-              const _Header(),
-              Expanded(
-                child: _PropertyColumn(
-                  cellBuilder: widget.cellBuilder,
-                  viewId: widget.dataController.viewId,
-                ),
+        child: ListView(
+          children: [
+            // const SizedBox(height: 100),
+            // const Divider(height: 1.0),
+            IntrinsicHeight(
+              child: Row(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                mainAxisSize: MainAxisSize.min,
+                children: [
+                  Flexible(
+                    flex: 4,
+                    child: _PropertyColumn(
+                      cellBuilder: widget.cellBuilder,
+                      viewId: widget.dataController.viewId,
+                    ),
+                  ),
+                  const VerticalDivider(width: 1.0),
+                  Flexible(
+                    child: _RowOptionColumn(
+                      viewId: widget.dataController.viewId,
+                      rowId: widget.dataController.rowId,
+                    ),
+                  ),
+                ],
               ),
-            ],
-          ),
+            ),
+            const Divider(height: 1.0),
+            const SizedBox(height: 200)
```

Comment:
This doesn't seem to be doing anything useful on behalf of the user. Should it be removed?

Reply:
I left this part and the other one above the `IntrinsicHeight` so as to demonstrate how the layout of the page could be done (with the future banner at the top, and the rich text area at the bottom) using a `ListView`. Without it, there wouldn't be a need to use `IntrinsicHeight` at all. What do you think?

Reply:
A comment above these widgets would be nice if that's the case. However, I would still like for these widgets to be removed since they're not doing anything :)

Reply:
Done!